core/sdk-core/src/main/java/software/amazon/awssdk/core/client/config/SdkClientOption.java

```diff
@@ -60,9 +61,30 @@ public final class SdkClientOption<T> extends ClientOption<T> {
     /**
      * @see ClientOverrideConfiguration#retryStrategy()
      */
-    @SuppressWarnings("rawtypes")
     public static final SdkClientOption<RetryStrategy> RETRY_STRATEGY = new SdkClientOption<>(RetryStrategy.class);
 
+    /**
+     * The internal retry strategy set by the customer. This is likely only useful within configuration
```

Comment:
The usage of the word `internal` here is confusing to me. Can we say `The retry strategy set by the customer [where/when]` instead?

Reply:
I updated the docs, e.g.,

```java
    /**
     * The retry mode set by the customer using {@link ClientOverrideConfiguration.Builder#retryStrategy(RetryMode)}. This is
     * likely only useful within configuration classes, and will be converted into a {@link #RETRY_STRATEGY} for the SDK's
     * runtime.
     *
     * @see ClientOverrideConfiguration#retryMode()
     */
```

Let me know if you have a better wording that I can use.

Reply:
I like this wording, thanks for updating.